BouncyCastle build warnings

When running `dotnet restore` on latest branch, seeing a lot of NU1701 errors related to `BouncyCastle`. Since this is a .NET core  / .net standard project, should we be using [Portable.BouncyCastle](https://www.nuget.org/packages/Portable.BouncyCastle) instead?

```
❯ dotnet restore
  Determining projects to restore...
/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.Core/WalletConnectSharp.Core.csproj : warning NU1701: Package 'BouncyCastle 1.8.9' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project. [/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.sln]
/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.NEthereum/WalletConnectSharp.NEthereum.csproj : warning NU1701: Package 'BouncyCastle 1.8.9' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project. [/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.sln]
/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.Desktop/WalletConnectSharp.Desktop.csproj : warning NU1701: Package 'BouncyCastle 1.8.9' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project. [/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.sln]
/repos/nft/deps/WalletConnectSharp/Examples/console/WalletConnectSharp.Examples.csproj : warning NU1701: Package 'BouncyCastle 1.8.9' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project. [/repos/nft/deps/WalletConnectSharp/WalletConnectSharp.sln]
```

--- a/WalletConnectSharp.Core/WalletConnectSharp.Core.csproj
+++ b/WalletConnectSharp.Core/WalletConnectSharp.Core.csproj
@@ -22,7 +22,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="BouncyCastle" Version="1.8.9" />
+      <PackageReference Include="Portable.BouncyCastle" Version="1.9.0" />
       <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
       <None Include="..\resources\icon.png" Pack="true" PackagePath="\" />
     </ItemGroup>

